xml/System/Char.xml

```diff
@@ -1067,10 +1067,7 @@ When a managed <xref:System.Char> type, which is represented as a Unicode UTF-16
         <summary>Indicates whether a specified Unicode character is categorized as a control character.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
```

Comment:
Since this paragraph applies to both overloads, a better option than deleting it is to keep this instance and delete the remaining two. That allows a Remarks section to continue to appear in the mini-TOC, where it's more likely to be noticed. (Sections of the documentation for individual overloads do not appear in the mini-TOC.)